pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
Yeah, I see your point. It's a good thing to set once for all a convention for all NGOs and group those profiles. It's also good to maintain a consistency between the profile name and the suffix in the release file name (I guess you expect something like `sigmah-hi-2.0.war` or something like this).

May I suggest `sigmah-ngo-*` as a prefix for all those profiles? So we could have `sigmah-ngo-handicap-international` or `sigmah-ngo-hi`.